graphqlapi/README.md

```diff
@@ -15,46 +16,39 @@ To build the GraphQL plugin individually, run the command:
 
 ### Using GraphQL
 
-By default, a GraphQL endpoint is started and will generate a GraphQL schema per 
-keyspace. You need at least one user-defined keyspace in your database to get 
-started.  
-
-# GraphQL API for Apache Cassandra
+By default, a GraphQL endpoint is started and will generate a GraphQL schema per keyspace. You need
+at least one user-defined keyspace in your database to get started.
 
 ### Using the Playground
 
-The easiest way to get started is to use the built-in GraphQL playground. GraphQL
-playground can be downloaded as a standalone app [here](https://github.com/prisma-labs/graphql-playground)
-and can then be accessed by going to http://localhost:8080/graphql/{keyspace}. 
-Once in the playground, you can create new schema and interact with the GraphQL APIs.
+The easiest way to get started is to use the built-in GraphQL playground and can then be accessed by
+going to <http://localhost:8080/graphql/{keyspace}>. Once in the playground, you can create new
+schema and interact with the GraphQL APIs.
 
 ### Path Layout
 
 By default, the server paths are structured to provide:
-* `/graphql-schema`: An API for exploring and creating schema, in
-database terminology this is know as: Data Definition Language (DDL). In
-Cassandra these are the queries used to create, modify, drop keyspaces and
-tables e.g. `CREATE KEYSPACE ...`, `CREATE TABLE ...`, `DROP TABLE ...`.
-* `/graphql/<keyspace>`: An API for querying and modifying your Cassandra
-tables using GraphQL fields.
 
-#### Keyspaces
+* `/graphql-schema`: An API for exploring and creating schema, in database terminology this is known
+  as: Data Definition Language (DDL). In Cassandra these are the queries used to create, modify,
+  drop keyspaces and tables e.g. `CREATE KEYSPACE ...`, `CREATE TABLE ...`, `DROP TABLE ...`.
+* `/graphql/<keyspace>`: An API for querying and modifying your Cassandra tables using GraphQL
+  fields.
 
-For each keyspace created in your Cassandra schema, a new path is created under
-the root `graphql-path` (default is: `/graphql`). For example, a path
-`/graphql/library` is created for the `library` keyspace when it is added to
-the Cassandra schema.
+#### Keyspaces
 
+For each keyspace created in your Cassandra schema, a new path is created under the
+root `graphql-path` (default is: `/graphql`). For example, a path
```

Comment:
This seems to imply that the `/graphql` prefix is configurable; currently it isn't.